packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -176,13 +181,19 @@ async function createWalletAddress(
     const walletAddress = await WalletAddress.query(
       deps.knex
     ).insertGraphAndFetch({
+      tenantId: options.tenantId,
```

Comment:
I think we would need the gets and update to also take in tenantId in the query, right?

Reply:
I don't think so, since you cannot move a wallet between tenants? Unless that is something we want? @mkurapov